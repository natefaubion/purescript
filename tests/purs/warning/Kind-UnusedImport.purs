-- @shouldWarnWith UnusedImport
module Main where

import Prelude (Unit, unit, pure)
import Effect (Effect)
import Type.RowList (RowList)

main :: Effect Unit
main = pure unit
