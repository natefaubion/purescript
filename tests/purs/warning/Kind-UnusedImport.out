Error found:
in module [33mMain[0m
at tests/purs/warning/Kind-UnusedImport.purs:6:1 - 6:35 (line 6, column 1 - line 6, column 35)

  The import of [33mType.RowList[0m is redundant


See https://github.com/purescript/documentation/blob/master/errors/UnusedImport.md for more information,
or to contribute content related to this error.

